src/style/app-fixed.module.css

```diff
@@ -801,13 +801,58 @@
 .modalHeader {
   border: none;
   padding-bottom: 0;
-  background-color: var(--modalHeader-bg) !important;
+  background-color: #f5f5f5 !important;
   padding: 1rem 1rem;
-  color: var(--modalHeader-color);
+  color: #4b5563 !important;
```

Comment:
Use variable names for colors. We will need this for our eventual migration to dark/light mode

There are elements in the CSS at the top of this file that you can use as examples

Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue` 

Use a variable name not the hex/rgba code (Use 8 digit HTML codes for transparency if necessary). Ensure that the variable name you choose describes the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`